src/library_html5.js

```diff
@@ -1865,6 +1865,12 @@ var LibraryJSEvents = {
     return GL.currentContext ? GL.currentContext.handle : 0;
   },
 
+  emscripten_webgl_get_drawing_buffer_size: function(contextHandle, width, height) {
+    var GLContext = GL.getContext(contextHandle).GLctx;
+    {{{ makeSetValue('width', '0', 'GLContext.drawingBufferWidth', 'i32') }}};
+    {{{ makeSetValue('height', '0', 'GLContext.drawingBufferHeight', 'i32') }}};
```

Comment:
Likewise, if `width` or `height` pointer is null, this should not write to the pointers, but would be good to return `{{{ cDefine('EMSCRIPTEN_RESULT_INVALID_PARAM') }}}`.